src/IssueTable.gs

```diff
@@ -128,10 +127,10 @@ function IssueTable(sheet, initRange, data) {
             break;
           case (key.hasOwnProperty('epic') && key.epic === true):
             if (key.value != 'n/a') {
-              if(undefined == epicField || epicField.usable === false || epicField.label_key == null) {
-                key.value = '=HYPERLINK("' + key.link + '"; "' + key.value + '")';
-              } else {
+              if (EpicField.isUsable()) {
                 key.value = '=HYPERLINK("' + key.link + '"; JST_EPICLABEL("' + key.value + '"))';
+              } else {
+                key.value = '=HYPERLINK("' + key.link + '"; "' + key.value + '")';
```

Comment:
Using the EpicField model rather than direct access to the UserStorage. I've manually tested it but not added unit testing in this file (too much work for now!)